README.md

```diff
@@ -102,6 +102,21 @@ Sample setting:
 ```
 "checker-framework.languageserver_repo": "checker-framework-languageserver"
 ```
```

Comment:
I suggest adding a "Requirements" section, which specifies that nodejs and yarn are required.